### ISSUE TITLE
A11y: Promotion submit button has no accessible name or visible label

### ISSUE THREAD

**Issue by shopwareBot** [shopware/platform#5213]
Currently, VoiceOver reads the promotions submit button as ‘button’. 

All controls need to have a visible label to act as a trigger for speech input users:
[https://www.w3.org/WAI/WCAG22/Understanding/label-in-name.html|https://www.w3.org/WAI/WCAG22/Understanding/label-in-name.html|smart-link] 

All content that is not text needs to have a text alternative. It should match the visible label.
[https://www.w3.org/WAI/WCAG22/Understanding/non-text-content.html|https://www.w3.org/WAI/WCAG22/Understanding/non-text-content.html|smart-link] 

Which can also double as instructions
[https://www.w3.org/WAI/WCAG22/Understanding/labels-or-instructions.html|https://www.w3.org/WAI/WCAG22/Understanding/labels-or-instructions.html|smart-link] 

*Steps to Reproduce:*

* Open your storefront in *Safari*
* Add an item to your cart
* Add a promotion
* Launch VoiceOver
* Navigate using Ctrl + Option + arrow keys to the submit button

!Screenshot 2024-10-04 at 18.22.59 (967d89a2-b6ea-4ecd-ba01-8ecf35c77f7d).png|width=370,height=58,alt="Screenshot 2024-10-04 at 18.22.59.png"!

**Comment by shopwareBot** [comment]
[public] Shopware automated response: This issue is linked to the internal issue: **NEXT-38712**



[created from NEXT-38712, comment 505559]

**Comment by shopwareBot** [comment]
[public] 

 > Status Change: **Open**

 This issue is ready for implementation.

 

[created from NEXT-38712, comment 510611]

**Comment by shopwareBot** [comment]
[public]

 > Status Change: **In Progress**

 Work on this issue has started. It is now being implemented.

 

[created from NEXT-38712, comment 510617]

**Comment by shopwareBot** [comment]
[public]

 > Status Change: **In Review**

 Work on this ticket is complete and is now pending review.



[created from NEXT-38712, comment 510623]

**Comment by shopwareBot** [comment]
[public]

 > Status Change: **Resolved**

 The ticket has been successfully completed; the solution is implemented and verified.



[created from NEXT-38712, comment 511393]

**Comment by shopwareBot** [comment]
[public] 

 > FixVersion Change: **6.6.9.0**

 This issue will be / was solved in Shopware version ***6.6.9.0***.



[created from NEXT-38712, comment 511394]
